cyclus_input_gen/from_pris.py

```diff
@@ -353,28 +351,27 @@ def reactor_render(reactor_data, output_file, is_cyborg=False):
                 country=data['country'].decode('utf-8'),
                 reactor_name=name,
                 type=reactor_type,
-                assem_size=523.4,
+                assem_size=446.0,
```

Comment:
reason for this change?

Reply:
this is the spec used in the paper. The previous assembly size was wrong.